src/common.css

```diff
@@ -173,21 +173,21 @@
     white-space: nowrap;
 }
 
-.ui-select-bootstrap .ui-select-choices-row>a:hover, .ui-select-bootstrap .ui-select-choices-row>a:focus {
+.ui-select-choices-row>a:hover, .ui-select-choices-row>a:focus {
     text-decoration: none;
     color: #262626;
     background-color: #f5f5f5;
 }
 
-.ui-select-bootstrap .ui-select-choices-row.active>a {
+.ui-select-choices-row.active>a {
     color: #fff;
     text-decoration: none;
     outline: 0;
     background-color: #428bca;
 }
 
-.ui-select-bootstrap .ui-select-choices-row.disabled>a,
-.ui-select-bootstrap .ui-select-choices-row.active.disabled>a {
+.ui-select-choices-row.disabled>a,
+.ui-select-choices-row.active.disabled>a {
```

Comment:
You mentioned in your PR that you only tested Bootstrap, removing the CSS class namespace would very likely have impact on the other themes, so please test them as well, seems likely this could cause a regression.